onnx/backend/test/case/node/blackmanwindow.py

```diff
@@ -37,9 +39,9 @@ def export() -> None:
         a2 = 0.08
         y = a0
         y += a1 * np.cos(
-            2 * 3.1415 * np.arange(0, size, 1, dtype=np.float32) / (size - 1)
+            2 * math.pi * np.arange(0, size, 1, dtype=np.float32) / (size - 1)
         )
         y += a2 * np.cos(
-            4 * 3.1415 * np.arange(0, size, 1, dtype=np.float32) / (size - 1)
+            4 * math.pi * np.arange(0, size, 1, dtype=np.float32) / (size - 1)
```

Comment:
I think the output.pb will need to be updated accordingly. (`3.141592653589793` vs original `3.1415`) Please rerun test generation.

Reply:
Done